README.md

```diff
@@ -61,15 +63,15 @@ def post_inference(prediction, metadata):
 
 ### Step 3: Deploy to AWS
 
-Deploying to AWS is as simple as running `cortex deploy` from your CLI. `cortex deploy` takes the declarative configuration from `cortex.yaml` and creates it on the cluster. Behind the scenes, Cortex containerizes the model, makes it servable using TensorFlow Serving, exposes the endpoint with a load balancer, and orchestrates the workload on Kubernetes.
+`cortex deploy` takes the declarative configuration from `cortex.yaml` and creates it on the cluster.
 
 ```bash
 $ cortex deploy
 
 deployment started
 ```
 
-You can track the status of a deployment using `cortex get`. The output below indicates that one replica of the API was requested and one replica is available to serve predictions. Cortex will automatically launch more replicas if the load increases and spin down replicas if there is unused capacity.
+You can track the status of a deployment using `cortex get`.
```

Comment:
Maybe add something along the lines of the last two sentences here after showing the output? (the sentences I'm referring to: `The output below indicates that one replica of the API was requested and one replica is available to serve predictions. Cortex will automatically launch more replicas if the load increases and spin down replicas if there is unused capacity.`